SDWebImage/SDWebImageWebPCoder.m

```diff
@@ -329,6 +333,28 @@ - (nullable UIImage *)sd_rawWebpImageWithData:(WebPData)webpData {
     return image;
 }
 
+// Create and return the correct colorspace by checking the ICC Profile
+- (nonnull CGColorSpaceRef)sd_colorSpaceWithDemuxer:(nonnull WebPDemuxer *)demuxer CF_RETURNS_RETAINED {
```

Comment:
Satisfy `Create Rule` may better.

Reply:
This method is create method...I place the `CF_RETURNS_RETAINED`, so that the caller should use `CFRelease` on that `CGColorSpace` object.